### PR TITLE
These are the updates for the init.d script to work.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

File: src/files/config/application.properties [hunk 1]
```diff
@@ -1,10 +1,4 @@
...
 # NEED TO SPECIFY THESE
-
-#spring.data.mongodb.host=localhost
-#spring.data.mongodb.port=12345
-#spring.data.mongodb.username=mongo_user
-#spring.data.mongodb.password=mongo_pass
-
-#or
 #spring.data.mongodb.uri=mongodb://mongo_user:mongo_pass@localhost:12345/db
+#logging.file=/var/log/repose-phone-home.log
```
Comment (adrianjgeorge) [inline src/files/config/application.properties]: is this intentionally commented out? This seems like one that is safe to leave on as the default

Comment (wdschei) [inline src/files/config/application.properties]: It was, but leaving it as the default sounds like a better idea.

**Comment by adrianjgeorge** [comment]
it looks like you got rid of sourcing any external options and the file that was in place for that, but you left the copy in the gradle script. Oversight, or did i miss where the new script was still sourcing it?

**Comment by wdschei** [comment]
Is it this line:
 - https://github.com/rackerlabs/repose-phone-home-service/blob/WDS_TweaksInFlight/src/files/inits/repose-phone-home-debian.init#L39

**Comment by adrianjgeorge** [comment]
yep, i missed it, then you need to not remove that empty file i had. i suppose we could put a comment in it saying it's for that purpose

**Comment by wdschei** [comment]
So should add a `src/files/config/repose-phone-home` to the project that just has a comment in it?

Does this add a file even if it doesn't exist in the project?
 - https://github.com/rackerlabs/repose-phone-home-service/blob/master/build.gradle#L95-L98

**Comment by adrianjgeorge** [comment]
i think it breaks if it's not there. i had the file before, but you removed it for some reason, probably because you thought it was left over from other changes.

**Comment by wdschei** [comment]
Cool. Adding it back now.

**Pull request merged**